src/server/server.babel.js

```diff
@@ -1,21 +1,3 @@
-const ignorePatterns = [
```

Comment:
piping is broken & i'm not sure why. When it auto-restarts, i get `EADDRESSINUSE` almost always. Also, it only worked in development, and we should be debugging server code in production.